resources/conf/config.toml.template

```diff
@@ -63,6 +63,26 @@ artifactsDirectory = "/home/wso2/artifacts"
   # Optional path to the private key for Consul communication. If this is set, then you need to also set certFile
   keyFile = "/home/wso2/security/truststore/consul/local-dc-client-consul-0-key.pem"
 
+# Configuration related to the repository where the api artifacts are stored
+[adapter.sourceControl]
+  # Enable/Disable Source Control for API Artifacts
+  enabled = true
+  # Time interval (in seconds) in which the Adapter should fetch updates from the remote repository
+  pollInterval = 30
+  # Time interval (in seconds) in which the adapter should retry fetching artifacts from the remote repository
```

Comment:
This is only happening at the startup of the adapter right? IF that is the case, let's mention that it is happening at the startup.

Reply:
Yes, retrying happens only on startup

Reply:
Fixed with cdeb393